cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedStmt.qll

```diff
@@ -933,6 +954,7 @@ class TranslatedCatchByTypeHandler extends TranslatedHandler {
       result = this.getParameter().getFirstInstruction(kind)
       or
       kind instanceof ExceptionEdge and
+      not kind.(ExceptionEdge).isSEH() and
```

Comment:
Like earlier comments: I don't think we need to hardcode the assumption that C++ and SEH exceptions don't mix. So I suggest we just delete this restriction.

Reply:
Fixed